src/page.js

```diff
@@ -6,6 +6,28 @@ export default class Page {
         this.target = 'page$' + pageId;
         this.phantom = phantom;
     }
+
+    on(event, runOnPhantom, callback) {
```

Comment:
Can you add doc here? Also, why is the use case of `runOnPhantom`? Isn't it the same thing as `.property` if it is true.